x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -180,44 +197,57 @@ function showResults(resp, action) {
 
 export async function cloneJob(jobId) {
   try {
-    const job = await loadFullJob(jobId);
-    if (job.custom_settings && job.custom_settings.created_by) {
+    const [cloneableJob, originalJob] = await Promise.all([
+      loadJobForExport(jobId),
+      loadFullJob(jobId, false),
+    ]);
+    if (cloneableJob !== undefined && originalJob?.custom_settings?.created_by !== undefined) {
       // if the job is from a wizards, i.e. contains a created_by property
       // use tempJobCloningObjects to temporarily store the job
-      mlJobService.tempJobCloningObjects.job = job;
+      // cloneableJob.custom_settings.created_by = originalJob?.custom_settings?.created_by;
```

Comment:
I guess this commented out line can be deleted?

Reply:
Removed here https://github.com/elastic/kibana/pull/88898/commits/9f446c20b3bf866b4a215e41b56b491bd6a27e7c